Mathlib/Data/Nat/GCD/Basic.lean

```diff
@@ -186,6 +228,24 @@ theorem coprime_mul_left_add_left (m n k : ℕ) : coprime (n * k + m) n ↔ copr
   rw [coprime, coprime, gcd_mul_left_add_left]
 #align nat.coprime_mul_left_add_left Nat.coprime_mul_left_add_left
 
+
+@[simp]
+theorem coprime_self_sub_left (m n : ℕ) (h : m ≤ n) : coprime (n - m) m ↔ coprime m n := by
```

Comment:
```suggestion
theorem coprime_sub_self_left (m n : ℕ) (h : m ≤ n) : coprime (n - m) m ↔ coprime m n := by
```
I would probably put `sub` first here, since it appears earlier in the goal.  If you like this change, the following lemmas can follow the same convention.

Reply:
I essentially followed your suggestion, except keeping the `left` with the name.

All arguements `m n` made implicit now, also above.